services/editor/src/pages/context/components/FixedKeys/FixedKeysList/FixedKey/FixedKey.js

```diff
@@ -61,8 +61,11 @@ const EditableKey = ({ keyPath, remote, local, onChange, autofocus }) => (
       disabled={local === undefined}
     />
     {remote !== undefined && remote !== local ? (
-      <div className="initial-value" title={remote}>
-        {remote}
+      <div
```

Comment:
There is a lot of conditions around `local`, `remote`, `null` and `undefined`.
Consider declare variables such as `hasChanged`, `keyValue`, `initialKeyValue` and calculate them in the beginning of render function.